packages/v4/src/content/design-guidelines/components/badge/badge.md

```diff
@@ -2,4 +2,11 @@
 id: Badge
 section: components
 ---
-A **badge** is used to annotate other information like a label or an object name. Badges are typically used to reflect a count, e.g. number of object, number of events, number of unread, etc.
+A **badge** is used to annotate other information like a label or an object name. 
+
+## Usage
+
+Badges are typically used to reflect a count, like number of object, number of events and number of unread, for example.
```

Comment:
typos: change "object" to "objects"; "unread" to "unread items"

Reply:
Updated!